src/test/ajaxWrapper.test.js

```diff
@@ -119,4 +133,64 @@ describe("ajaxWrapper", function () {
 
   });
 
+  describe("on POST request", function () {
+
+    it("sends the correct payload", function (done) {
+      var response = {body: ""};
+      var request = {method: null};
+      var payload = {"key": "value"};
+
+      this.server.on("request", function (req) {
```

Comment:
I think this should go into the HttpServer.js directly to make it reusable. Also it is really useful.